dali/dfu/dfurun.cpp

```diff
@@ -1354,20 +1356,14 @@ class CDFUengine: public CInterface, implements IDFUengine
                             Owned<IPropertyTree> plane = getStoragePlane(clusterName);
                             if (plane)
                             {
-                                if (plane->hasProp("@defaultSprayParts"))
+                                if (plane->hasProp("@defaultSprayParts") && destination->getNumPartsOverride()==0)
```

Comment:
Is there any reason this functionality is in a isContainerized() block?
In bare-metal it getting the plane and @defaultSprayParts should work too.

NB: subDirPerFilePart is Containerized only.